extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayProcessor.java

```diff
@@ -196,9 +195,9 @@ ServiceStartBuildItem createBeansAndStartActions(FlywayRecorder recorder,
         return migrationLocations;
     }
 
-    private List<String> discoverApplicationMigrations(Collection<String> locations) throws IOException, URISyntaxException {
+    private Set<String> discoverApplicationMigrations(Collection<String> locations) throws IOException, URISyntaxException {
         try {
-            List<String> applicationMigrationResources = new ArrayList<>();
+            Set<String> applicationMigrationResources = new HashSet<>();
```

Comment:
nice one @geoand

Reply:
Yeah, because we do resource discovery at build time we need this to avoid duplicated being passed to Flyway

Reply:
Will this fix make it into 1.5.3?

Reply:
There won't be a 1.5.3 most likely. It should hopefully be in the upcoming 1.6.0.CR1

Reply:
any ETA on 1.6.0.CR1.

Reply:
Should be in a few days